internal/api/handlers.go

```diff
@@ -382,3 +433,17 @@ var GeneratePassword = func(length int) (string, error) {
 	}
 	return string(b), nil
 }
+
+// Helper function to generate a JWT
+func generateJWT(username, jwtSecret string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
```

Comment:
i think we mentioned using HMAC here.

Reply:
isn't `SigningMethodHS256` from hmac? or what do you mean?

Reply:
oh it is, nevermind